Rewrite INSTALL instructions for python novices (#4750)
scikit-image/scikit-image: CONTRIBUTING.txt

Build environment setup
-----------------------

> Please reffer to the Installation instructions to install scikit-image

Reviewer: Also, reffer -> refer

> Please reffer to the Installation instructions to install scikit-image

Reviewer: I *think* this should work?

```suggestion
Please refer to `Installing scikit-image`_ for development installation instructions.
```

Testing
-------

> See the testing section of the Installtion guide.

Reviewer 2: Typo!

Reviewer: I pushed a fix!
